Stripe/STPPaymentIntentParams.m

```diff
@@ -0,0 +1,74 @@
+//
+//  STPPaymentIntentParams.m
+//  Stripe
+//
+//  Created by Daniel Jackson on 7/3/18.
+//  Copyright © 2018 Stripe, Inc. All rights reserved.
+//
+
+#import "STPPaymentIntentParams.h"
+#import "STPPaymentIntent+Private.h"
+
```

Comment:
I find it useful to include NS_ASSUME_NONNULL_BEGIN/END in the .m files as well. It makes sure your implementation matches your interface!